base/common/ya.make

```diff
@@ -33,18 +33,18 @@ PEERDIR(
 )
 
 SRCS(
-    argsToConfig.cpp
-    coverage.cpp
     DateLUT.cpp
     DateLUTImpl.cpp
+    JSON.cpp
```

Comment:
Are the order changes significant, or is it a missing sort in generation script?